sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Tests/InitializationTests.cs

```diff
@@ -401,7 +401,6 @@ public static void EvaluateLoggerProvider(IServiceProvider serviceProvider, bool
 
                     var secondProcessor = valueField.GetValue(secondNode);
                     Assert.NotNull(secondProcessor);
-                    Assert.Contains("BatchLogRecordExportProcessor", secondProcessor.GetType().Name);
```

Comment:
This test needs to be refactored, its based on reflection and follows ordering on processors.